packages/nitro-protocol/src/contract/transaction-creators/erc20-asset-holder.ts

```diff
@@ -46,7 +46,7 @@ export function createDepositTransaction(
   expectedHeld: string,
   amount: string
 ): ethers.providers.TransactionRequest {
-  const data = Erc20AssetHolderContractInterface.functions.deposit.encode([
+  const data = Erc20AssetHolderContractInterface.encodeFunctionData('deposit', [
```

Comment:
Worryingly, this error was not flagged by `yarn tsc` until I upgraded typescript. 😟